omnibus/config/software/system-probe.rb

```diff
@@ -62,4 +61,6 @@
   end
 
   copy 'pkg/ebpf/c/COPYING', "#{install_dir}/embedded/share/system-probe/ebpf/"
+
+  link "#{install_dir}/bin/agent/agent", "#{install_dir}/embedded/bin/system-probe"
```

Comment:
This is working in locally but does omnibus link strip/fixup the install_dir path ?
I would say the correct syntax should be
`link ../../bin/agent/agent, "#{install_dir}/embedded/bin/system-probe"`

Reply:
The `link` function for omnibus is kind of broken : it checks that the source of the symlink exists, but it does so by changing the current directory to install_dir and check the relative path (instead of using the `dirname` of the destination of the symlink)